### ISSUE TITLE
Broken installer on Ubuntu 17.04

### ISSUE THREAD

**Issue by rissicay** [elixir-lang/elixir#5990]
### Environment

* Elixir & Erlang versions (elixir --version): trying to install latest from elixir-lang instructions
* Operating system: Ubuntu 17.04

### Current behavior

```
sudo apt-get install esl-erlang
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 esl-erlang : Depends: libssl0.9.8 but it is not installable
              Depends: libwxbase2.8-0 but it is not installable
              Depends: libwxgtk2.8-0 but it is not installable
              Recommends: erlang-mode but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

I think the dependencies need to be updated..

**Comment by josevalim** [comment]
Thank you for the report but we are not the ones that maintain distributions (there are too many of them for us to be the ones doing it). Please ping "@erlangsolutions" on Twitter or reach for contact on their package page.

**Comment by cdesch** [comment]
@rissicay Did you find a solution for this?

**Comment by muhajirframe** [comment]
same question @rissicay

**Comment by cdesch** [comment]
there is a package that you can so `sudo apt-get install elixir` but it appears to be an older package.  I used that package the manually upgraded it to elixir version 1.5.1.

**Comment by muhajirframe** [comment]
@cdesch I ended up using docker image. But I notice that this commnad 
```
wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
sudo dpkg -i erlang-solutions_1.0_all.deb
```
Doesn't add the ubuntu version to repository, but debian version. You may want to try to `alternatively` methods from https://www.erlang-solutions.com/resources/download.html . Though it doesn't seems provide the version for ubuntu 17.04. 

And finally, you can download .deb file from there and install it on your favorite machine :)

**Comment by cdesch** [comment]
I'll give it a try. thanks

**Comment by rissicay** [comment]
I didn't find a solution for this, I too just used docker.

**Comment by category** [comment]
I'm getting the same issue.
Here's a workaround.
Download Erlang OTP from the[ downloads section](https://www.erlang-solutions.com/resources/download.html), then install.
Then download and run precompiled Elixir from the [relevant section](https://elixir-lang.org/install.html#precompiled-package).

**Comment by cdesch** [comment]
I can't recall exactly what I did back then, but I believe the installation errored out in the terminal, although the installation actually completed fine. AFAIK, I was able to run Elixir fine for the project I was working on, although I may not (Most Likely Not) have used every module in Elixir/Erlang. I'm not sure if one of those modules may have been what caused the installation to fail and through not using them I may have been able to scrape by. 

@category I'll give your method a shot.
